### ISSUE TITLE
progress bar for data module

### ISSUE THREAD

**Issue by akeshavan** [nipy/dmriprep#26]
```python
from dmriprep.data import get_dataset
get_dataset('path/to/output/dir') # needs a progress bar in bash and also on notebook?
```

* also would be nice if pathlib made the folder for you if it doesn't already exist `mkdir -p`
* maybe remove `Site-SI` from output folder structure

**Comment by akeshavan** [comment]
also for the HBN data, add the function that moves the T1.nii.gz in derivatives to the freesurfer folder.

**Comment by richford** [comment]
Closed by #38
